### PR TITLE
Image::getMesh returns a mesh instead of vtkPolyData

### PR DESCRIPTION
**Pull request by archanasri**

### CODE REVIEW

File: Libs/Image/Image.h [hunk 1]
```diff
@@ -244,7 +244,7 @@ class Image
   TransformPtr createRigidRegistrationTransform(const Image &target, float isoValue = 0.0, unsigned iterations = 20);
 
   /// creates a vtkPolyData for the given image
-  static vtkSmartPointer<vtkPolyData> getPolyData(const Image& image, PixelType isoValue = 0.0);
+  static Mesh getMesh(const Image& image, PixelType isoValue = 0.0);
```
Comment (cchriste) [inline Libs/Image/Image.h:247]: Unless there is a specific reason for this function to be static, let's simply make it a member of Image.

**Review by cchriste — CHANGES_REQUESTED** [review]
Looks good except for the (perhaps) unnecessarily static function in Image. 

I also wonder if we might add `using Transform = vtkSmartPointer<vtkMatrix4x4>;` to Shapeworks.h.

File: Libs/Image/Image.h (hunk 1 above)
Comment (archanasri) [reply Libs/Image/Image.h]: Done

**Review by cchriste — APPROVED** [review]
An even more worthwhile fix than expected, simplifying the file layouts and compilation.

**Pull request merged**